providers/pool.rb

```diff
@@ -188,7 +188,7 @@ def configure
     configure_application_pool(is_new_auto_start, "autoStart:#{new_resource.auto_start}")
```

Comment:
Also line 134 contains 'APPPOOL/@PipelineMode'.capitalize

irb(main):001:0> 'APPPOOL/@PipelineMode'.capitalize
=> "Apppool/@pipelinemode"

That's probably not what you wanted.